### DISCUSSION TITLE
The given trained BDD100K model mismatches with the config file qdtrack-frcnn_r50_fpn_12e_bdd100k.py

### DISCUSSION THREAD

**Discussion by dwluo** [ethvis/qd-track#124] [General]
Config file: https://github.com/SysCV/qdtrack/blob/master/configs/bdd100k/qdtrack-frcnn_r50_fpn_12e_bdd100k.py
checkpoint: the BDD100K model link on webpage https://github.com/SysCV/qdtrack

I tried to load the trained BDD100K model and perfrom inference on some image data. However, it seems like that the released BDD100K checkpoint did not match with the given config file.

Can anybody check on this one?

**Comment by YtJin-git** [comment]
Have you solved this problem?

**Comment by hoqolo** [comment]
I have the same question

**Comment by Raspberry-beans** [comment]
Anyone who has solved this issue?
